### ISSUE TITLE
Very low contrast ratio and very thin fonts in theme MODXRE2

### ISSUE THREAD

**Issue by sashabeep** [modxcms/evolution#1079]
# Subnav item

![2016-12-08 17 33 44](https://cloud.githubusercontent.com/assets/5689796/21013881/0d4f27e8-bd75-11e6-9d22-f57176e30de7.png)
Test output: 9.0:1
![2016-12-08 17 32 22](https://cloud.githubusercontent.com/assets/5689796/21013882/0d525058-bd75-11e6-8486-24bb83647ff7.png)

MODXRE2
![2016-12-08 17 33 37](https://cloud.githubusercontent.com/assets/5689796/21013916/2c3b514a-bd75-11e6-96e6-3d792882e8e2.png)
Test output: 2.9:1
![2016-12-08 17 31 30](https://cloud.githubusercontent.com/assets/5689796/21013917/2c41ad38-bd75-11e6-9885-5a7a86567fec.png)

# Documents tree
![2016-12-08 17 37 49](https://cloud.githubusercontent.com/assets/5689796/21013994/92cbd128-bd75-11e6-8540-a8bbf3ecdf4f.png)
![2016-12-08 17 37 45](https://cloud.githubusercontent.com/assets/5689796/21013993/92c88bf8-bd75-11e6-9464-29d163692851.png)

MODXRE2
![2016-12-08 17 38 30](https://cloud.githubusercontent.com/assets/5689796/21014005/9c208b92-bd75-11e6-9a52-fcd1e1e6f8a4.png)
![2016-12-08 17 38 22](https://cloud.githubusercontent.com/assets/5689796/21014006/9c244b06-bd75-11e6-95c3-3cef008cdbde.png)

## Functional items also very frustruating, no accent even for the hover
![2016-12-08 17 44 37](https://cloud.githubusercontent.com/assets/5689796/21014205/8990bb04-bd76-11e6-9d84-165bf2f89ffb.png)

## Horizontal dividers is missing on the first tab of settings
![2016-12-08 17 47 10](https://cloud.githubusercontent.com/assets/5689796/21014300/e33d45b4-bd76-11e6-820a-077d83d63dbc.png)

And many other places, for example unreadable font weight in section headline, compared with with previous version MODXRE
![2016-12-08 17 40 45](https://cloud.githubusercontent.com/assets/5689796/21014089/02e740c8-bd76-11e6-9d3d-51df2f8f9329.png)
![2016-12-08 17 40 38](https://cloud.githubusercontent.com/assets/5689796/21014090/02e8f396-bd76-11e6-8756-fc42fcd2a295.png)

https://www.w3.org/TR/WCAG/#visual-audio-contrast

After 1.2 update old themes is unuseable because of lacking some icons, search box is not properly positioned etc.

**Comment by dmi3yy** [comment]
about contrast +1 need change for more contrasts  

--  
Lukianenko Dmytro
Web developer  
Site:       http://dmi3yy.com
Skype:   dmi3yy
Phone:  +38 (091) 357-10-80


четверг, 8 декабря 2016 г. в 16:51, Sasha Beep написал:

**Comment by fourroses666** [comment]
+1

I have made a little bit restyled custom version which is a little bit better to read.
Removed Forgot password on Login.
Also changed TinyMCE font and icon sizes.

- Upload files
- On Config tab > Interface > Content Management Systeem theme > select `Custom-RE2`
- On same tab scroll down to `TimeMCE 4 settings` and select `evo-custom`

[manager-theme.zip](https://github.com/modxcms/evolution/files/639771/manager-theme.zip)

**Comment by dmi3yy** [comment]
i think better with smalles padding like  
http://take.ms/PUGkt


--  
Lukianenko Dmytro
Web developer  
Site:       http://dmi3yy.com
Skype:   dmi3yy
Phone:  +38 (091) 357-10-80


четверг, 8 декабря 2016 г. в 17:12, Marc написал:

**Comment by dmi3yy** [comment]
and more contrast to theme  
if not 2 window  i not see diferents )


--  
Lukianenko Dmytro
Web developer  
Site:       http://dmi3yy.com
Skype:   dmi3yy
Phone:  +38 (091) 357-10-80


четверг, 8 декабря 2016 г. в 17:18, Lukianenko Dmytro написал:

**Comment by sashabeep** [comment]
Another one style variation
[Custom-RE3.zip](https://github.com/modxcms/evolution/files/641896/Custom-RE3.zip)

**Comment by fourroses666** [comment]
Done some re-restyling. 
Better reading and added multilang html lang= like Yama added here;
But not sure if it works for the other languages..
https://github.com/modxcms/evolution/issues/1085

See old:
![old](https://cloud.githubusercontent.com/assets/523389/21055134/8b622318-be30-11e6-9b97-b3f90964022d.jpg)

New:
![new](https://cloud.githubusercontent.com/assets/523389/21055143/920233c0-be30-11e6-938f-629fe4991f83.jpg)

[Custom-RE4.zip](https://github.com/modxcms/evolution/files/642602/Custom-RE4.zip)

**Comment by Nicola1971** [comment]
..just because i always working on manager things :-D

# About old themes

> After 1.2 update old themes is unuseable because of lacking some icons, search box is not properly positioned etc.

### MODxRE1.2
https://github.com/Nicola1971/MODxRE1.2

![re1 2](https://cloud.githubusercontent.com/assets/7342798/21054684/e94aca9a-be2e-11e6-9206-4601ab2c52ef.png)

### MODxCarbon 1.2
https://github.com/Nicola1971/MODxCarbon1.2

![carbon1 2](https://cloud.githubusercontent.com/assets/7342798/21054719/feac9148-be2e-11e6-9457-6350fb087542.png)


# About MODxRE2

For small colours customizations on MODxRE2

you can try this plugin:

### MODxRE2Colors (beta)
https://github.com/Nicola1971/MODxRE2Colors-Plugin
![re2colors](https://cloud.githubusercontent.com/assets/7342798/21054789/4ca55600-be2f-11e6-8a9f-918a19910e18.png)

# Alternative Themes

For a complete different manager style for  Evo1.2 

### MODxFLAT 3
https://github.com/Nicola1971/MODxFLAT
![modx-flat](https://cloud.githubusercontent.com/assets/7342798/21054867/975f9138-be2f-11e6-84cc-0bee19de28a8.png)

![flat-el](https://cloud.githubusercontent.com/assets/7342798/21055191/b99e60de-be30-11e6-8dbc-e02b787e9ec2.png)


and his plugin
### MODxFlatColors
https://github.com/Nicola1971/MODxFlatColors-Plugin
![flatcolors](https://cloud.githubusercontent.com/assets/7342798/21054915/c13e1b14-be2f-11e6-9f69-4c86292f1f16.png)

![flatcolors-el](https://cloud.githubusercontent.com/assets/7342798/21055186/b487c66c-be30-11e6-888b-f005cfbc001f.png)
